### PR TITLE
Update file-name-transact-sql.md

### PR DESCRIPTION
**Pull request by 0x7FFFFFFFFFFFFFFF**
Add important information about the function.

### CODE REVIEW

File: docs/t-sql/functions/file-name-transact-sql.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ dev_langs:
 # FILE_NAME (Transact-SQL)
 [!INCLUDE [SQL Server Azure SQL Managed Instance](../../includes/applies-to-version/sql-asdbmi.md)]
 
-This function returns the logical file name for a given file identification (ID) number.  
+This function returns the logical file name for a given file identification (ID) number, within the context of current database.
```
Comment (rwestMSFT) [inline docs/t-sql/functions/file-name-transact-sql.md]:
```suggestion
This function returns the logical file name for a given file identification (ID) number, in the context of the current database.
```

**Comment by Court72** [comment]
@markingmyname

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@Court72 You can sign this off at your convenience.

**Comment by Court72** [comment]
Due to the upcoming Ignite conference, all merging and publishing is on hold until the morning of 11/15.

This PR is ready for merge on 11/15.

#label: "event-triaged-to-10AM"
#label: "missed-content-complete"

**Pull request merged**